x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -191,6 +191,21 @@ export const allowedExperimentalValues = Object.freeze({
    * Expires: on Feb 20, 2024
    */
   jsonPrebuiltRulesDiffingEnabled: true,
+  /*
```

Comment:
Probably need to update with main as it shouldn't be showing all of these?

Reply:
@michaelolo24 , actually during a merge conflict, the positions changed for existing flags.  If you scrolls down a little, it is matching with `main` just fine.